WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -1,126 +1,140 @@
 import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let navBarTitle = "my-site-navigation-bar"
-    static let blogTable = "Blog Details Table"
-    static let removeSiteButton = "BlogDetailsRemoveSiteCell"
-    static let activityLogButton = "Activity Log Row"
-    static let jetpackScanButton = "Scan Row"
-    static let jetpackBackupButton = "Backup Row"
-    static let postsButton = "Blog Post Row"
-    static let mediaButton = "Media Row"
-    static let statsButton = "Stats Row"
-    static let peopleButton = "People Row"
-    static let settingsButton = "Settings Row"
-    static let domainsButton = "Domains Row"
-    static let createButton = "floatingCreateButton"
-    static let ReaderButton = "Reader"
-    static let switchSiteButton = "SwitchSiteButton"
-    static let dashboardButton = "Home"
-    static let segmentedControlMenuButton = "Menu"
-    // "Free To Paid Plans" Card
-    static let freeToPaidPlansCardId = "dashboard-free-to-paid-plans-card-contentview"
-    static let freeToPaidPlansCardHeaderButton = "Free domain with an annual plan"
-    // "Pages" Card
-    static let pagesCardId = "dashboard-pages-card-frameview"
-    static let pagesCardHeaderButton = "Pages"
-    static let pagesCardMoreButton = "More"
-    static let pagesCardCreatePageButton = "Create another page"
-    // "Activity Log" Card
-    static let activityLogCardId = "dashboard-activity-log-card-frameview"
-    static let activityLogCardHeaderButton = "Recent activity"
-}
-
 /// The home-base screen for an individual site. Used in many of our UI tests.
 public class MySiteScreen: ScreenObject {
-    public let tabBar: TabNavComponent
```

Comment:
removing `tabBar` from individual screens. we can access tab bar separately from the test. adding it on every screen is an extra check for the first element on the screen _and_ the tab bar element which can make the test slow, without adding much value to the test (as we would know if there are issues on the tab bar from the tab bar tests)